packages/mongo/collection.js

```diff
@@ -210,15 +210,15 @@ Mongo.Collection = function (name, options) {
         return self.findOne(id);
       },
     });
-
-    if (!ok)
-      throw new Error("There is already a collection named '" + name + "'");
+    
+    // If ok is false, that means we already registered the store for this collection name, but
```

Comment:
I think the best option would be a `warnIfCollectionExists` property on the Collection constructor and then a warning on the console if it set.